lib/rubocop/config_loader.rb

```diff
@@ -60,7 +60,7 @@ def create_config(hash, path)
       end
 
       def add_missing_namespaces(path, hash)
-        hash.keys.each do |k|
+        hash.clone.each_key do |k|
           q = Cop::Cop.qualified_cop_name(k, path)
           next if q == k
 
```

Comment:
(Anecdotally: I think ActiveSupport added a monkey patch for this. `Hash#transform_keys`.)